relayer/src/chain/counterparty.rs

```diff
@@ -399,6 +399,29 @@ pub fn unreceived_acknowledgements_sequences(
         .map_err(Error::relayer)
 }
 
+/// This method expects an [`IdentifiedChannelEnd`] plus a pair of
```

Comment:
I would flip the order of these first two sentences; start off with what this method does, and then mention what the method expects.